pkg/config/application_ecs.go

```diff
@@ -34,9 +38,21 @@ func (s *ECSApplicationSpec) Validate() error {
 }
 
 type ECSDeploymentInput struct {
+	// The name of network configuration file placing in application directory.
+	VpcConfigurationFile string `json:"vpcConfigurationFile"`
+	// The Amazon Resource Name (ARN) that identifies the cluster.
+	ClusterArn string `json:"clusterArn"`
+	// The launch type on which to run your task.
+	// If a launchType is specified, the capacityProviderStrategy parameter must be omitted.
```

Comment:
nits, should add a link to AWS docs
https://docs.aws.amazon.com/AmazonECS/latest/developerguide/launch_types.html